tls/s2n_early_data.c

```diff
@@ -130,6 +132,34 @@ S2N_RESULT s2n_early_data_accept_or_reject(struct s2n_connection *conn)
     return S2N_RESULT_OK;
 }
 
+int s2n_config_set_server_max_early_data_size(struct s2n_config *config, uint32_t max_early_data_size)
+{
+    POSIX_ENSURE_REF(config);
+    config->server_max_early_data_size = max_early_data_size;
```

Comment:
Can the `max_early_data_size` input be zero?

Reply:
No reason why not. That would disable early data.